### ISSUE TITLE
Error Module 'CausalSelfAttention' has no attribute 'bias' :

### ISSUE THREAD

**Issue by pallingua** [suno-ai/bark#493]
I followed the instructions, installed bark locally,  then ran the example, I got  Error Module 'CausalSelfAttention' has no attribute 'bias' :  error.

Module 'CausalSelfAttention' has no attribute 'bias' :
  File "C:\Users\xxx\AppData\Roaming\Python\Python310\site-packages\bark\model.py", line 84
            # manual implementation of attention
            att = (q @ k.transpose(-2, -1)) * (1.0 / math.sqrt(k.size(-1)))
            att = att.masked_fill(self.bias[:,:,FULL_T-T:FULL_T,:FULL_T] == 0, float('-inf'))
                                  ~~~~~~~~~ <--- HERE
            att = F.softmax(att, dim=-1)
            att = self.attn_dropout(att)
